docs/src/main/paradox/java/http/routing-dsl/directives/alphabetically.md

```diff
@@ -1,154 +1,153 @@
-|@ref[withSettings](basic-directives/withSettings.md#withsettings-java)                                                                       | Runs its inner route with the given alternative `RoutingSettings`                                                                                                                                                   |
-|@ref[withSizeLimit](misc-directives/withSizeLimit.md#withsizelimit-java)                                                                     | Applies request entity size check                                                                                                                                                                                   |
-|@ref[withoutRequestTimeout](timeout-directives/withoutRequestTimeout.md#withoutrequesttimeout-java)                                          | Disables @ref[request timeouts](../../common/timeouts.md#request-timeout) for a given route.                                                                                                                   |
-|@ref[withoutSizeLimit](misc-directives/withoutSizeLimit.md#withoutsizelimit-java)                                                            | Skips request entity size check                                                                                                                                                                                     |
+|Directive                                                                                                            | Description                                                                                                                                                                                                         |
+|---------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|@ref[authenticateBasic](security-directives/authenticateBasic.md)                                                    | Wraps the inner route with Http Basic authentication support using a given `Authenticator<T>`                                                                                                                       |
+|@ref[authenticateBasicAsync](security-directives/authenticateBasicAsync.md)                                          | Wraps the inner route with Http Basic authentication support using a given `AsyncAuthenticator<T>`                                                                                                                  |
```

Comment:
I was not able to find any trailing spaces here nor for the two other cases you pointed out. How can I reveal them? `git show` doesn't display a red box at the end of the line as it normally does for trailing spaces.